### ISSUE TITLE
InfluxDB.Client.Core.Exceptions.InfluxException : Object of type 'NodaTime.Instant' cannot be converted to type 'System.Nullable`1[System.DateTime]'.

### ISSUE THREAD

**Issue by nzabsodislab** [influxdata/influxdb-client-csharp#280]
Hello,

__Steps to reproduce:__

Query to measurement:
```
queryApi.QueryAsync<Measurement>("from(bucket: ""Measurement"")
                   |> range(start: 1971-09-18T12:00:00.000000000Z)
                   |> filter(fn: (r) => r[""_measurement""] == ""measurement"")
                |> yield()")
```

Same query in editor works fine.

__Expected behavior:__
All results from selected date

__Actual behavior:__
 Object of type 'NodaTime.Instant' cannot be converted to type 'System.Nullable`1[System.DateTime]'.
  ----> System.ArgumentException : Object of type 'NodaTime.Instant' cannot be converted to type 'System.Nullable`1[System.DateTime]'.

__Specifications:__
 - Client Version: 3.2.0
 - InfluxDB Version: 2.1.1
 - Platform:

**Comment by bednar** [comment]
Hi @nzabsodislab,

thanks for using our client.

Can you share how looks like your `Measurement` class?

Regards

**Comment by nzabsodislab** [comment]
Good evening,

```
[InfluxDB.Client.Core.Measurement("measurement")]
    public class Measurement
    {
        [Column(IsTimestamp = true)]
        public DateTime? Time { get; set; }

        [Column("area_id", IsTag = true)]
        public string AreaId { get; set; }

        [Column("system_id", IsTag = true)]
        public string SystemId { get; set; }

        [Column("name")]
        public string Name { get; set; }

        [Column("value")]
        public double? Value { get; set; }
        
        [Column("unit")]
        public string MeasurementUnit { get; set; }
    }
```

**Comment by bednar** [comment]
@nzabsodislab thanks for detail info

I have prepared a fixed version within #281. Development version of the client will be available after #281 acceptance, stable version in regular release cycle.
